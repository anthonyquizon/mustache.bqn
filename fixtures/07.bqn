data⇐{
  a⇐⟨{
    b⇐⟨{d⇐1}, {d⇐2}⟩,
    c⇐⟨{d⇐"a"}, {d⇐"b"}⟩
  },{
    b⇐⟨{d⇐3}, {d⇐4}⟩,
    c⇐⟨{d⇐"c"}, {d⇐"d"}⟩
  }⟩
}
