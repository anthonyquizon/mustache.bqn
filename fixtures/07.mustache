<x>
    {{#a}}
        <a>
            {{#b}}
                <b>
                    <d>{{d}}</d>
                </b>
            {{/b}}
            {{#c}}
                <c>
                    <d>{{d}}</d>
                </c>
            {{/c}}
        </a>
    {{/a}}
</x>
