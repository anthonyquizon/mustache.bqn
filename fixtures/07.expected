<x>
        <a>
                <b>
                    <d>1</d>
                </b>
                <b>
                    <d>2</d>
                </b>
                <c>
                    <d>a</d>
                </c>
                <c>
                    <d>b</d>
                </c>
        </a>
        <a>
                <b>
                    <d>3</d>
                </b>
                <b>
                    <d>4</d>
                </b>
                <b>
                    <d>5</d>
                </b>
                <c>
                    <d>c</d>
                </c>
                <c>
                    <d>d</d>
                </c>
                <c>
                    <d>e</d>
                </c>
        </a>
</x>
